app/(app)/profile/edit/page.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+
+import Breadcrumb from "@/components/ui/breadcrumb";
+import { useAuth } from "@/hooks/useAuth";
+import { useEffect, useState } from "react";
+import { CloudUpload } from "lucide-react";
+import Image from "next/image";
+import { Skeleton } from "@/components/ui/skeleton";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { useTranslations } from "next-intl";
+
+const BREADCRUMB_ROUTES = [
+  {
+    name: "Profile",
+    href: "/profile",
+  },
+  {
+    name: "Edit Profile",
+  },
+];
+
+const EditProfilePage = () => {
+  const { state } = useAuth();
+  const { userInfo, isLoading } = state;
+  const { picture, name, email } = userInfo || {};
+  const [selectedPicture, setSelectedPicture] = useState<string | undefined | null>(null);
+  const [inputName, setInputName] = useState<string | undefined>("");
+  const [inputEmail, setInputEmail] = useState<string | undefined>("");
+  const t = useTranslations("profile");
+
+  useEffect(() => {
+    if (!isLoading) {
+      setSelectedPicture(picture);
+      setInputName(name);
+      setInputEmail(email);
+    }
+  }, [isLoading]);
+
+  const handleImageChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setSelectedPicture(reader.result as string);
+      };
+      reader.readAsDataURL(file);
+    }
+  };
+
+  return (
+    <main className="px-[20px] mt-4 xl:mt-8 max-w-[935px] w-full">
+      <div className="mb-0 xl:mb-2">
+        <Breadcrumb routes={BREADCRUMB_ROUTES} />
+      </div>
+      <h1 className="unbound-font text-[20px] font-medium text-black xl:text-[34px] xl:font-bold xl:mb-6 mb-4">
+        {t("editProfile")}
+      </h1>
+      <div
+        className="flex flex-col items-center px-6 pt-6 xl:pb-[41px]
+        pb-[13px] rounded-[8px] border-[#E0E0E0] border-[1px]"
```

Comment:
Create `--light-gray: 0 0% 88%;` in globals and tailwind config to create `border-light-gray`